packages/core/src/utils/graphFacade.ts

```diff
@@ -60,6 +60,18 @@ export class GraphFacade {
     await this.graphite.scheduler.drain();
   }
 
+  public async createRawNode(raw: string, { type, path }: Pick<ISourceNode, 'type' | 'path'>) {
+    this.graphite.graph.addNode({
+      category: NodeCategory.Source,
+      type,
+      language: filenameToLanguage(path),
+      path: '/' + path,
```

Comment:
Note: I hadd to add the `/` otherwise Graphite complains because the source node is not an absolute path. However, we need the path (in this case an URL) there.